### DISCUSSION TITLE
UI Power Units Mismatch

### DISCUSSION THREAD

**Discussion by signifywinter** [netbox-community/netbox#8969] [General]
Has anyone else noticed that there is a mismatch between the power units on a Device page vs. the Power Ports page?  VA do not always equal W.

![device-page](https://user-images.githubusercontent.com/102368995/160138366-4f95edea-80bd-47b8-b42c-8e02c8f525c8.png)

![pp-page](https://user-images.githubusercontent.com/102368995/160138368-5e88f3af-81e9-40d0-acf8-2618c7076c95.png)
 page?

**Comment by chicks-net** [comment]
> The volt-ampere is dimensionally equivalent to the [watt](https://en.wikipedia.org/wiki/Watt) (in [SI units](https://en.wikipedia.org/wiki/SI_units), 1 V⋅A = 1 W).

from [wikipedia on Volt-Ampere](https://en.wikipedia.org/wiki/Volt-ampere)

**Comment by signifywinter** [comment]
I get that, but they are not the same thing.  VA is apparent power vs. W
being real power.

What's more, certain equipment's VA rating is not always equal to W rating,
which is why this mismatch jumped out at me.

https://www.eaton.com/us/en-us/skuPage.5PX2200IRT.html
https://www.cyberpowersystems.com/product/ups/pfc-sinewave/cp1500pfclcd/

I only called this out because the same number has different units on two
pages.  Not a huge deal, but something I noticed.

**Reply by jcollie** [reply]
I'd agree that NetBox should be consistent on the units displayed. What would REALLY be nice is to be able to track BOTH real & apparent power.

**Reply by iunderwood** [reply]
VA = W when the power factor is 1, but that is extremely rare.

The representation of power utilization should be consistent throughout, but if we are going to tinker with VA, we'll need to at least be able to calculate or store the power factor.
